localisation/00_wc_artifact_localisation.csv

```diff
@@ -65,6 +65,10 @@ twilight_canticle;Twilight Canticle;;Loblied des Zwielichts;;;;;;;;;;;x
 twilight_canticle_desc;This book contains inscrutable knowledge about the Void and the Old Gods. The pages are made of the flesh and skin of faithful followers, who gave their lives for the sake of the will of their lords.;;Dieses Buch enth�lt unergr�ndliches Wissen �ber die Leere und die alten G�tter. Die Seiten sind aus der Haut und dem Fleisch treuer Anh�nger gemacht, die ihr Leben f�r den Willen ihrer Herren gegeben haben.;;;;;;;;;;;x
 xalatath;Xal'atath, Blade of the Black Empire;Xal'atath, lame de l'Empire noir;Xal'atath, Klinge des schwarzen Imperiums;;;;;;;;;;;x
 xalatath_desc;Nobody knows exactly when this blade was created, but according to legend it was used for sacrifices in ancient times. It drips with void magic and its quiet whispering attacks those with weak minds.;;Niemand wei� genau, wann diese Klinge geschaffen wurde, aber den Sagen zufolge wurde sie offenbar in alten Zeiten f�r Opferungen verwendet. Sie trieft vor Leeren-Magie und ihr leises Fl�stern bef�llt diejenigen mit einem schwachen Verstand.;;;;;;;;;;;x
+twin_blades_of_azzinoth;Twin Blades of Azzinoth;;Kriegsgleven von Azzinoth;;;;;;;;;;;
+twin_blades_of_azzinoth_desc;These twin-blades formerly belonged to Azzinoth, the leader of the doomguard, who was slain by Illidan 10.000 years ago. They are usually connected, but can be seperated and used as two weapons.;;Diese Doppelklingen geh�rten einst Azzinoth, dem Anf�hrer der Schreckenswache, der vor 10.000 Jahren von Illidan get�tet wurde. Normalerweise sind sie verbunden, k�nnen aber getrennt und als zwei Waffen benutzt werden.;;;;;;;;;;;
+gorehowl;Gorehowl;;Blutschrei;;;;;;;;;;;
+gorehowl_desc;Massive axe wielded by Grommash Hellscream of the Warsong Clan. It is permanently stained with the blood of the warrior's fallen enemies. Small holes and notches that dot the blade make it �sing� when swung.;;Eine riesige von Grommash Hellscream des Warsong-Klans benutzte Axt. Sie ist permanent mit dem Blut gefallener Opfer des Kriegers beschmiert. Kleine L�cher und Einkerbungen an der Klinge lassen die Axt �singen� wenn sie geschwungen wird.;;;;;;;;;;;
```

Comment:
```suggestion
gorehowl_desc;Massive axe wielded by Grommash Hellscream of the Warsong Clan. It is permanently stained with the blood of the warrior's fallen enemies. Small holes and notches that dot the blade make it "sing" when swung.;;Eine riesige von Grommash Hellscream des Warsong-Klans benutzte Axt. Sie ist permanent mit dem Blut gefallener Opfer des Kriegers beschmiert. Kleine L�cher und Einkerbungen an der Klinge lassen die Axt �singen� wenn sie geschwungen wird.;;;;;;;;;;;
```
IIRC, English not often use these quotation marks.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/71102729-659f2a00-21d2-11ea-8a5f-21fbc24b2a95.png)

</details>

Reply:
🤔 

![image](https://user-images.githubusercontent.com/46576860/71111436-80759c80-21da-11ea-85b9-267740aa7998.png)

Reply:
@MemeWizard this error with quotation marks will be resolved in the next localisation crusade, already committed there.